definitions/artifacts/azure-storage-account.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "azure-storage-account"
+  },
+  "type": "object",
+  "title": "Azure Storage Account",
+  "description": "",
+  "additionalProperties": false,
+  "required": [
+    "data",
+    "specs"
+  ],
+  "properties": {
+    "data": {
+      "title": "Artifact Data",
+      "type": "object",
+      "required": [
+        "infrastructure",
+        "authentication",
+        "security"
+      ],
+      "properties": {
+        "infrastructure": {
+          "title": "Infrastructure configuration",
+          "type": "object",
+          "required": [
+            "ari"
+          ],
+          "properties": {
+            "ari": {
+              "$ref": "../types/azure-resource-id.json"
+            }
+          }
+        },
+        "authentication": {
+          "title": "Authentication credentials",
+          "type": "object",
+          "required": [
+            "azure-storage-account-endpoint",
```

Comment:
What are your thoughts on `endpoint` and `connection_string` here? Other places where we've used `hostname` we don't prefix it w/ the type, since we have that context from the artifact name.

Reply:
The regexes used in the types are unique to Azure, and I don't think we want non-Azure bundle to mistakenly try and use this artifact, as that will fail.

Reply:
@WillBeebe what do you think?

Reply:
I really do think these should be `endpoint` and `connection_string`

Reply:
Agree. Its scoped to azure. Key names should also be underscore not hyphens

Reply:
To be clear the type file names are fine, the properties should be underscored and a more succinct name like endpoint or connection_string. The code rendering this artifact already knows it’s an azure storage account by referencing this main definition file.

Reply:
Agreed on `endpoint` and `connection_string`. Just by being in this artifact its clear they belong to an azure storage account 👍

Reply:
Gotcha, I misunderstood and assumed the suggestion was to rename the types themselves. I do like this suggestion though. Thanks @WillBeebe !